doctr/models/recognition/sar.py

```diff
@@ -134,7 +137,10 @@ def call(
             logits = tf.concat([logits, glimpse], axis=-1)
             # shape (N, rnn_units + 1) -> (N, num_classes + 1)
             logits = self.output_dense(logits, **kwargs)
+            # update symbol with predicted logits for t+1 step
+            symbol = tf.argmax(logits, axis=-1)
```

Comment:
Could you elaborate a bit more in the PR description what was not done correctly and how this fixes it please? If there is some TF documentation reference, please add it as well :pray: